src/lib/auth0-provider.ts

```diff
@@ -0,0 +1,5 @@
+'use client';
+
+import { Auth0Provider } from '@auth0/auth0-react';
+
+export default Auth0Provider;
```

Comment:
We need `use client` so the Auth0Provider works but we need to extract it so we can set metadata in the root page.